src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -467,6 +471,14 @@ private void filmSuchen2(String urlFilm_, String thema, String titel, long dauer
 			}
 		}
 
+        private String addMissingHttpPrefixIfNecessary(String aUrl) {
+            if(aUrl.startsWith("//")) {
+                aUrl = TEXT_START_HTTP + ":" + aUrl;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "aUrl". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)